### PR TITLE
chore: add tracks to jaffle shop

### PR DESCRIPTION
**Pull request by rephus**
Closes: 

### Description:

Screenshot from 2024-04-11 10-06-32

### Reviewer actions

- [ ] I have manually tested the changes in the preview environment
- [ ] I have reviewed the code
- [ ] I understand that "request changes" will block this PR from merging

### CODE REVIEW

**Comment by rephus** [comment]
I only included data from the last 6 montsh to reduce the size of the file (dbt seed)

File: examples/full-jaffle-shop-demo/dbt/dbt_project.yml [hunk 1]
```diff
@@ -30,3 +30,6 @@ seeds:
     raw_plan:
       +column_types:
         metadata: jsonb
+    raw_tracks:
+      +column_types:
+        timestamp: timestamp
```
Comment (rephus) [inline examples/full-jaffle-shop-demo/dbt/dbt_project.yml:35]: otherwise it adds the field as a string ...

**Review by TuringLovesDeathMetal — APPROVED** [review]
Looks good to me!

**Review by magnew — APPROVED** [review]
Looks great. I still might add the 1 event per hour table, but we don't need to hold this review up.

**Comment by rephus** [comment]
> Looks great. I still might add the 1 event per hour table, but we don't need to hold this review up.

Sure, you we can add it on a separate PR.

**Comment by rephus** [comment]
Note: we'll have to run seed on all staging warehouses, to make this available

**Pull request merged**